easybuild/easyblocks/t/tbb.py

```diff
@@ -162,7 +162,6 @@ def install_step(self):
             self.libpath = os.path.join(self.libpath, libdir)
             self.log.debug("self.libpath: %s" % self.libpath)
             # applications go looking into tbb/lib so we move what's in there to libs
-            # and symlink the right lib from /tbb/libs/intel64/... to lib
```

Comment:
@mboisson Why remove this comment here, there's symlinking being done below?

The comment should state it's creating a `tbb/lib` symlink though?